cli/aiconfig-editor/README.md

```diff
@@ -0,0 +1,40 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+
+## Getting Started
+
+First, run the development server:
+
+```bash
```

Comment:
I know this is just the default from the quickstart, but noting we should standardize on yarn to match the rest of the repo